### PR TITLE
Fixes for the creation wizard and bonus

### PR DESCRIPTION
**Pull request by FinalAngel**
This centralises the wizards javascript into one separate file which is more favourable than splitting up.

Also fixes an issue where wizard does not show up on welcome screen.

Also sneaks in a lighter gray variant for disabled button state:
image

### CODE REVIEW

**Comment by mkoistinen** [comment]
LGTM for both the lighter button and the JS moves

**Comment by mkoistinen** [comment]
LGTM!

File: cms/static/cms/sass/mixins/_buttons.scss [hunk 1]
```diff
@@ -68,4 +68,7 @@
             }
         }
     }
+    &.cms-btn-disabled {
+        color: $gray-lighter;
```
Comment (vxsx) [inline cms/static/cms/sass/mixins/_buttons.scss]: that's going to work only for default buttons, not for action / danger ones.
maybe it's better to fix line 63 in this file to be more transparent?

File: cms/static/cms/sass/mixins/_buttons.scss [hunk 2]
```diff
@@ -68,4 +68,7 @@
             }
         }
     }
+    &.cms-btn-disabled {
+        color: lighten($color, 50%);
```
Comment (vxsx) [inline cms/static/cms/sass/mixins/_buttons.scss]: @FinalAngel can you move it up to line 63? this one doesn't cover all cases and doesn't have the important function

**Comment by vxsx** [comment]
:+1:

**Pull request merged**